src/map/latent_effect_container.cpp

```diff
@@ -414,6 +424,16 @@ void CLatentEffectContainer::CheckLatentsEquip(uint8 slot)
                     m_LatentEffectList.at(i)->Deactivate();
                 }
                 break;
+            case LATENT_WEAPON_DRAWN_MP_OVER:
+                if (m_POwner->health.mp > m_LatentEffectList.at(i)->GetConditionsValue() && m_POwner->animation == ANIMATION_ATTACK)
+                {
+                    m_LatentEffectList.at(i)->Activate();
+                }
+                else
+                {
+                    m_LatentEffectList.at(i)->Deactivate();
+                }
+				break;
```

Comment:
almost!
rebase, then change this line to use spaces, amend last commit, force push